uaclient/conftest.py

```diff
@@ -99,7 +99,8 @@ def logging_sandbox():
 def FakeConfig(tmpdir):
     class _FakeConfig(UAConfig):
         def __init__(self, cfg_overrides={}, features_override=None) -> None:
-            cfg_overrides.update({"data_dir": tmpdir.strpath})
+            if not cfg_overrides or not cfg_overrides.get("data_dir"):
```

Comment:
I think we can only check for the `cfg_overrides.get("data_dir")` here. This just simplifies for the case where we have an empty dict or a dict without `data_dir` defined

Reply:
Makes sense.